crates/polars-plan/src/dsl/mod.rs

```diff
@@ -1198,6 +1198,9 @@ impl Expr {
     #[cfg(feature = "is_in")]
     pub fn is_in<E: Into<Expr>>(self, other: E, nulls_equal: bool) -> Self {
         let other = other.into();
+
+        // TODO: I believe this should be all_leaves_literal but this crashes
```

Comment:
@alexander-beedie After this PR gets merged, could you take a look at this and see why changing this to `all_leaves_literal` crashes `test_in_subquery`? It causes a `polars.exceptions.ColumnNotFoundError: ylFFDoaPFomSXxNoN`.